# Exemplos de Teste de Hipótese

## Teste Z para média

Em um anúncio, uma pizzaria alega que o tempo médio de entrega é inferior a 30 minutos. O gerente da pizzaria foi questionado sobre a veracidade da afirmação e por isso resolveu solicitar a um analista de dados que verifica-se a situação. O analista coletou aleatoriamente uma amostra com 36 tempos de entrega e observou uma média amostral de 28,5 minutos. Sabe-se, de estudos anteriores, que o desvio padrão da entrega é de 3,5 minutos. Há evidências suficientes para confirmar a alegação com um nível de significância de 1%.

<span style="color:red">H<sub>a</sub></span> < 30

<span style="color:blue">H<sub>0</sub></span> $\geqslant$ 30

n = 36 minutos

$\sigma$ = 3.5 minutos

$\alpha$ = 1% = 0.01

$\text{Z = }\frac{(36 - 30)}{\frac{3.5}{\sqrt{36}}}$

In [1]:
import math

Zo = (28.5 - 30)*math.sqrt(36)/3.5
print('Zobservado = ', Zo*(-1))

Zobservado =  2.5714285714285716


In [2]:
A1 = 0.5 - 0.01
print('Pesquisar na tabela área = ', A1)
print()

Pesquisar na tabela área =  0.49



**Z<sub>critico</sub> = 2.33**

**Z<sub>observado</sub> =  2.5714285714285716**

**Z<sub>observado</sub> > Z<sub>critico</sub> cai na área crítica então rejeitar H<sub>0</sub>**

Resposta:

**Há evidências que o tempo médio de entrega é inferior a 30 minutos, com um nível de significância de 1%.**

## Teste t Student para a média

Um vendedor de motocicleta afirma que o preço médio do modelo Honda CG Titan é de pelo menos R\\$ 16.500,00. Vôce suspeita da alegação e investiga o preço pela internet de uma amostra aleatória de 14 motocicletas do modelo, concluindo que o preço médio do estudo foi de R\\$ 15.700,00 e desvio padrão R\\$ 1250,00. Há evidêmcias suficientes para rejeitar a alegação do vendedor a um nível de significância de 5%? Suponha que a população seja normalmente distribuida.

<span style="color:blue">H<sub>0</sub></span> $\geqslant $ 16.500

<span style="color:red">H<sub>a</sub></span> $\lt$ 16500

n = 14 $\Longrightarrow $ n < 30 usar a distribuição t Student

gl = 14 - 1 = 13

$\bar{x} $ = 15.700

$\sigma$ = 1.250

$\alpha$ = 0,05

t<sub>crítico</sub> = -1,771


(15.700 - 16.500)$\sqrt{14}$/1250

In [3]:
to = (15700 - 16500)*math.sqrt(14)/1250
print('t observado = ', to)

t observado =  -2.394660727535322


**t<sub>critico</sub> = 1.771**

**t<sub>observado</sub> =  2.346**

**t<sub>observado</sub> > t<sub>crítico</sub> cai na região crítica então rejeito H<sub>0</sub>**

Resposta:

**Ao nível de significância de 5% não a evidência de que a média do preço é maior ou igual 16.500,00**

## Teste de Z para Proporção

Um pesquisador alega que menos de 20% dos adultos brasileiros alérgicos a um medicamento. Em uma amostra aleatória de 100 adultos, 15 disseram ter esse tipo de alergia. Teste a alegação do pesquisador com um nível de significância de 1%.

0n.p = 100 x 20 = 20

$\text{n(1- p)}\geqslant 5 = 100(1 -.20) = 80 \geqslant 30$

n.p $\geqslant$ 5 $\Longrightarrow$ posso usar a normal

n(1 - p) $\Longrightarrow$ 5 posso usara a normal

<span style="color:blue">H<sub>0</sub></span> $\geqslant $ 0.80

<span style="color:red">H<sub>a</sub></span> $\lt$  0,20

n = 100

$\hat{p} = \frac{15}{100} = 0.15$

$\text{Z = }\Large \frac{\hat{p} - p}{\sqrt{\frac{p(1 - p)}{n}}}$

In [4]:
q = 0.15 - 0.20
d = math.sqrt( ( 0.20 * (1 - 0.2 ) ) / 100 )
print('q = ', round(q, 4))
print('d = ', round(d, 4))
Z = q / d
print('Z critico = ', round(Z, 4))

q =  -0.05
d =  0.04
Z critico =  -1.25


Z<sub>critico</sub> = -2.325

Z<sub>observado</sub> = -1.25

Com um nível de significância de 1% não há evidências que menos de 20% dos adultos são alérgicos ao medicamento.


## Teste de hipótese para a diferença das médias

Hś uma crença que diz que o volume de venda médio (em R\\$) dos vendedores de uma empresa de SP é o mesmo que dos vendedores da filial do RJ. Teste ao nível de significância de 5% a vereacidade dessa afirmação.

|         SP         |         RJ         |
|:-------------------|:-------------------|
|      n = 100       |      n = 100       |
| $\bar{x}$ = 60.900 | $\bar{x}$ = 64.300 |
|     S = 12.000     |     S = 15.000     |


<span style="color:red">H<sub>0</sub></span> $\mu_{sp}\text{ = }\mu_{rj}$

<span style="color:blue">H<sub>a</sub></span> $\mu_{sp}\ \cong \mu_{rj}$

<span style="color:red">H<sub>0</sub></span> $\mu_{sp} - \mu_{rj} \text{ = 0}$

<span style="color:blue">H<sub>a</sub></span> $\mu_{sp} - \mu_{rj} \cong \text{ 0}$





$\Large \text{Z = }\frac{(x_{1} - x_{2}) - (\mu_{1} - \mu_{2})}{\sqrt{\frac{S_{1}^{2}}{n_{1}} + \frac{S_{2}^{2}}{n_{2}}}}$

$\Large \text{Z = }\frac{(60.900 - 64.300) - (0)}{\sqrt{\frac{\text{12.000}²}{100} + \frac{\text{15.000}²}{100}}}$

In [5]:
q = 60900 - 64300
d = math.sqrt( (12000**2 + 15000**2) / 100 )
print('q = ', q)
print('d = ', round(d, 4))
Zo = q / d
print('Zobservado = ', round(Zo, 2))

q =  -3400
d =  1920.9373
Zobservado =  -1.77


Teste dos lados a curva:

$\alpha = \Large \frac{\alpha}{2}$

$\alpha = \Large \frac{5}{2}$

$\alpha = 2.5 = 0.025$

Area = 0.5 - 0.025

Z<sub>critico</sub> = 1.96

In [6]:
A1 = 0.5 - 0.025
print('A1 = ', A1)
print('Zc = ', 1.96)
print('Zo = ', 1.77)

A1 =  0.475
Zc =  1.96
Zo =  1.77


Para um nível de significância de 5%, não pode se rejeitar que as médias de vendas de SP e RJ são iguais.